projects/docs-tools/src/modules/demo-page/demo-page.component.ts

```diff
@@ -74,7 +96,30 @@ export class SkyDocsDemoPageComponent
    * Specifies the local path to the module's source code. The value is relative to the root directory.
    */
   @Input()
-  public moduleSourceCodePath: string;
+  public set moduleSourceCodePath(path: string | undefined) {
+    this.#_moduleSourceCodePath = path;
+    this.testingSourceCodePath = path.replace('modules', 'testing');
+    this.#updateTypes();
+  }
+
+  public get moduleSourceCodePath(): string | undefined {
+    return this.#_moduleSourceCodePath;
+  }
+
+  /**
+   * Specifies the type definitions for the development tab.
+   */
+  public moduleTypeDefinitions: SkyDocsTypeDefinitions | undefined;
+
+  /**
+   * Specifies the local path to the testing source code. The value is relative to the root directory.
+   */
+  public testingSourceCodePath: string;
+
+  /**
+   * Specifies the type definitions for the testing tab.
+   */
+  public testTypeDefinitions: SkyDocsTypeDefinitions | undefined;
```

Comment:
I'd be consistent with the other properties and name this `testingTypeDefinitions`.

Reply:
Done